extensions/ql-vscode/package.json

```diff
@@ -2079,9 +2081,9 @@
     "gulp-replace": "^1.1.3",
     "gulp-typescript": "^5.0.1",
     "husky": "^9.1.5",
-    "jest": "^29.0.3",
-    "jest-environment-jsdom": "^29.0.3",
+    "jest": "^30.0.0-alpha.7",
     "jest-runner-vscode": "^3.0.1",
+    "jsdom": "^26.0.0",
```

Comment:
Looking at [`jest-environment-jsdom`](https://www.npmjs.com/package/jest-environment-jsdom?activeTab=code) it seems we're going `jsdom` version 20 to 26.

Looking in the [changelog](https://github.com/jsdom/jsdom/blob/main/Changelog.md) I can't immediately see any references to the fields that weren't defined and were causing errors, but it could be I missed it or it's from another updated dependency 🤷🏼

Reply:
I believe what we're looking for is in [version 23.1.0](https://github.com/jsdom/jsdom/blob/main/Changelog.md#2310): "Added an initial implementation of `ElementInternals`, including the `shadowRoot` getter and the string-valued ARIA properties." This is what was giving errors when using some `@vscode-elements/elements` components (after fixing a missing `CSSProperties.replaceSync` definition).